packages/suite-web-landing/components/Layout/index.tsx

```diff
@@ -4,7 +4,7 @@ import Translation from '../Translation';
 import { TrezorLogo, Button, colors, variables, Link } from '@trezor/components';
 
 const Layout = styled.div`
-    max-width: ${variables.SCREEN_SIZE.XL};
+    max-width: calc(${variables.SCREEN_SIZE.XL} + 80px);
```

Comment:
there's another layout width used in landing page vs figma design, because of that some texts were differently aligned and we've decided with brand team to change it due figma. I agree it could be another commit.